src/components/tx/SignOrExecuteForm/ExecuteForm.tsx

```diff
@@ -57,11 +56,10 @@ const ExecuteForm = ({
   const [executionMethod, setExecutionMethod] = useState(ExecutionMethod.RELAY)
 
   // SC wallets can relay fully signed transactions
-  const [walletCanRelay] = useWalletCanRelay(safeTx)
-
+  const [canRelay] = useWalletCanRelay(safeTx)
```

Comment:
I think this naming is more confusing now since its not about if the user can relay but if their wallet supports relaying.

Reply:
Renamed it to `canWalletRelay`.